AWSSDKSwiftCLI/Sources/AWSSDKSwiftCLI/Resources/Package.Base.swift

```diff
@@ -22,6 +22,7 @@ extension Target.Dependency {
     static var awsSDKEventStreamsAuth: Self { "AWSSDKEventStreamsAuth" }
     static var awsSDKHTTPAuth: Self { "AWSSDKHTTPAuth" }
     static var awsSDKIdentity: Self { "AWSSDKIdentity" }
+    static var awsSDKChecksums: Self { "AWSSDKChecksums" }
```

Comment:
Question: Is a separate module actually necessary for just a single extension with two methods?

Reply:
I wish more could be moved to AWSSDKChecksums but that's dependent on CRT HTTP Client being refactored in the future. I'd say even if its only two methods its good to have but I don't feel strongly. `aws-chunked` is pretty explicitly aws-only though and one of those methods adds it to the headers

Reply:
I see. We could keep it the way it is in this PR, or move it into AWSClientRuntime. For both cases we would revisit this down the line. I don't feel strongly about either option. Unless Josh has preference for one over the other I suppose we can keep `AWSSDKChecksums`.